docs/docs-content/integrations/k3s.md

```diff
@@ -0,0 +1,143 @@
+
+You can add cloud-init stages, which allow you to customize your instances declaratively. The cloud-init stages are exposed by [Kairos](https://kairos.io/docs/architecture/cloud-init/), an open source project. For more information, check out the [Cloud Init Stages](../clusters/edge/edge-configuration/cloud-init.md) reference.
+
+#### Palette Virtual Cluster
+Since the Palette Virtual Cluster is a virtual cluster that you are setting up inside another Kubernetes cluster, you can configure its pods and services differently than the host cluster. The kubeadm file you get includes parameters that offer you a higher degree of customization. 
+The following is a list of important parameters when you deploy a Palette Virtual Cluster cluster using K3s:
+
+|**Parameter**|**Description** |
+|-------------|----------------|
+|`enableHA`| Determines whether the control plane is deployed in high availability mode. If you set this parameter to true, make sure to adjust the number of replicas and use an external datastore. |
+|`defaultImageRegistry` | Specifies the default registry from which images are pulled. The value of this parameter will be prepended to all deployed virtual cluster images. If an image has already been deployed as part of the virtual cluster, the deployed images within the virtual cluster will not be rewritten.|
+|`sync` | Specifies which Kubernetes resources are synced between the virtual and host clusters. |
+|`storage` | Specifies storage settings such as persistence and PVC size. By default, storage of the virtual cluster uses the same storage class as the host cluster, but you can also optionally specify a different storage class. |
+|`ingress` | Configures the ingress resource that allows you to access the virtual cluster |
+
+### Usage
+
+K3s is available for Edge host deployments as well as Palette Virtual Clusters that you can create from cluster groups. Refer to the [Create an Infrastructure Profile](../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md) guide and the [Create and Manage Cluster Groups](https://docs.spectrocloud.com/clusters/cluster-groups/create-cluster-group#palette-virtual-cluster-configuration) guide for more information. 
+
+:::info
+In order to use K3s as part of an Edge deployment, you need to go through the EdgeForge process and specify K3s as your intended Kubernetes distribution when you build your OS image. For more information, refer to the [EdgeForge Workflow](https://docs.spectrocloud.com/clusters/edge/edgeforge-workflow/) guide. 
+:::
+
+#### Configure OIDC Identity Provider for Edge Deployments
+
+You can modify the kubeadm file to configure your Edge cluster to use an OIDC Identity Provider (IDP) for authentication. 
+You can use a custom third-party IDP such as Okta or you can use Palette as your IDP. 
+
+When you add the K3s pack to a cluster profile, Palette displays the OIDC IDP options listed below:
```

Comment:
```suggestion
When you add the K3s pack to a cluster profile, Palette displays OIDC IDP options, which you can configure in the Kubernetes layer. To learn about these options, refer to the [Palette eXtended Kubernetes - Edge](/integrations/kubernetes-edge/#configure-oidc-identity-provider) section.
```

Reply:
Instead of duplicating content, we typically document it in one place and link to it because it makes content management easier - this allows us to make updates in one place. All of the OIDC information that follows is documented in the Kubernetes pack docs (PXK and PXK-E). 

If the user _must_ configure Custom, you could tell them that and refer them to existing documentation.  

Note that the link above is not a relative path, and I'm not sure that's where you want to take your reader for Palette Virtual Cluster. OIDC is also documented here for Palette Virtual Clusters: https://docs.spectrocloud.com/clusters/palette-virtual-clusters/configure-oidc-virtual-cluster

Reply:
For this one, it is true that it is duplicating content, but i feel like users who are using this pack wouldn't want to go to another pack for information. I think this duplication is okay for now. Maybe in the future we can move both content to a separate page to something like "Configuring OIDC for Edge" under the edge documentation and link to that page then